SYCL/ESIMD/sycl_esimd_mix_check_build_opts.cpp

```diff
@@ -0,0 +1,148 @@
+//==--- sycl_esimd_mix_check_build_opts.cpp - DPC++ ESIMD on-device test ---==//
```

Comment:
according to my previous comment, this test is not needed

Reply:
Done.